app/decorators/service_provider_session_decorator.rb

```diff
@@ -48,4 +54,8 @@ def sp_return_url
   private
 
   attr_reader :sp, :view_context
+
+  def t(string, options = {})
```

Comment:
I think you can get around this "uncommunicative method name" reek by using `delegate :t, to: :view_context`